lib/Makefile

```diff
@@ -374,7 +378,7 @@ clean:
 #-----------------------------------------------------------------------------
 # make install is validated only for below listed environments
 #-----------------------------------------------------------------------------
-ifneq (,$(filter $(UNAME),Linux Darwin GNU/kFreeBSD GNU OpenBSD FreeBSD NetBSD DragonFly SunOS Haiku))
+ifneq (,$(filter $(UNAME),Linux Darwin GNU/kFreeBSD GNU OpenBSD FreeBSD NetBSD DragonFly SunOS Haiku AIX))
```

Comment:
tangentially related :
It seems we have the same list of `uname` identifiers in 3 different places (for the purpose of controlling `make install` compatibility).
We could probably consolidate this list in one place and distribute it, making its maintenance and evolution a bit easier.
cc @felixhandte .